specification/attributes/key_value_format.md

```diff
@@ -0,0 +1,32 @@
+# Key-Value Format
+
+Columns that provide Key-Value information are often used in place of separate columns for enumerating data which would be inherently sparse and/or without predetermined keys. This consolidates related information and provides more consistency in the schema. Key-value pairs are also referred to as name-value pairs, attribute-value pairs, or field-value pairs.
+
+All key-value related columns defined in the FOCUS specification MUST follow the key-value formatting requirements listed below. These columns SHALL NOT contain nested elements.
+
+## Attribute ID
+
+KeyValueFormat
+
+## Attribute Name
+
+Key-Value Format
```

Comment:
Should it be KeyValue Format? 

I ask because we also have Date/Time Format and it feels like it would be better to not have - or /. Otherwise we should pick one Applies here and for the top of the page header (see [here](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/blob/working_draft/specification/attributes/datetime_format.md))

Reply:
I like Keyvalue, DateTime....

Reply:
Wikipedia uses [key-value](https://en.wikipedia.org/wiki/Key%E2%80%93value_database) and [date/time](https://en.wikipedia.org/wiki/ISO_8601#EDTF) just to keep us on our toes 🙃 I believe "DateTime" is slightly different in that it refers to a datatype in a programming language. I vaguely recall looking that up when we went thru that one.

Reply:
To me, the Attribute Name is written in plain English as you would when using it in an email and Attribute ID is written in the way you'd use it in a system or reference it in a slack message with technical teams.

In English, it's very atypical to have a compound word written in Pascal case.

For now, we can leave it as key-value since we have it written as date/time in the working draft. If we want to change this, we should do both (and any similar examples we find) in the same PR.